lib/Yam/Agama/Pom/GrubMenuAnyPage.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles GRUB screen.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Agama::Pom::GrubMenuAnyPage;
```

Comment:
we have a different implementation on purpose to have fixed expecations for each product, for instance, check:
`lib/Yam/Agama/Pom/GrubMenuLeapPage.pm`

Reply:
The different approach was that I first aimed for having  the module that uses it in any test that uses `agama.yaml`. Then we changed that and focused on the USB boot failure. So now I modified the existing `GrubAgamaPage.pm` and put the code to boot from harddisk in there. VR's restartetd, still getting a blank screen in SLE 16, but that's another issue I guess.